.gitignore

```diff
@@ -18,3 +18,4 @@ Gemfile.lock
 Icon
 
 Thumbs.db
+_config.yml
```

Comment:
_config.yml shouldn't be ignored. It's what is used for site-wide variables like site.url

Reply:
On further discussion, we're ignoring _config for now to deal with local testing issues